### PR TITLE
Improved error handling when downloading CLI.

### PR DESCRIPTION
**Pull request by ArmandGrillet**
To close https://jira.mesosphere.com/browse/DCOS-21077 we have added an extra step checking for an existing `dcos` file when downloading the CLI through the DC/OS UI. With the change of path of curl's download, we now check that there is no existing `dcos` file or directory in `/usr/local/bin/` before downloading the new CLI.

### CODE REVIEW

File: src/js/components/modals/CliInstallModal.js [hunk 1]
```diff
@@ -74,11 +75,11 @@ class CliInstallModal extends React.Component {
 
     const instructions = [
       `[ -d /usr/local/bin ] || sudo mkdir -p /usr/local/bin`,
-      `curl ${downloadUrl} -o dcos`,
-      `sudo mv dcos /usr/local/bin`,
+      `[ -f /usr/local/bin/dcos ]`,
```
Comment (rgo3) [inline src/js/components/modals/CliInstallModal.js]: Missing `&&`?

Comment (ArmandGrillet) [reply src/js/components/modals/CliInstallModal.js]: They are added automatically (line 93).

**Comment by nLight** [comment]
@ArmandGrillet we now have two identical PRs this one and #3325 is this intentional?

**Comment by ArmandGrillet** [comment]
@nLight Closing this one, #3325 should be the one getting merged ASAP.

**Pull request closed without merging**